### PR TITLE
Update Connector Service chart and READMEs

### PR DESCRIPTION
**Pull request by janmedrek**
**Description**

Changes proposed in this pull request:

- README.md updated for chart and component
- introduced parameter for enabling / disabling tests

**Related issue(s)**

### CODE REVIEW

File: resources/application-connector/charts/connector-service/README.md [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
 - **internalAPIPort** - This port exposes the Connector Service within Kubernetes cluster. The default port is `8080`.
 - **namespace** - Namespace where Connector Service is deployed. The default Namespace is `kyma-integration`.
 - **tokenLength** - Length of registration tokens. The default value is `64`.
-- **tokenExpirationMinutes** - Time after which tokens expire and are no longer valid. The default value is `5` minutes.
-- **domainName** - Domain name of the cluster, used for generating URL. Default domain name is `.wormhole.cluster.kyma.cx`.
-- **certificateServiceHost** - Host at which this service is accessible, used for generating URL. Default host is `cert-service.wormhole.cluster.kyma.cx`.
+- **appTokenExpirationMinutes** - Time after which tokens for applications expire and are no longer valid. The default value is `5` minutes.
+- **runtimeTokenExpirationMinutes** - Time after which tokens for runtimes expire and are no longer valid. The default value is `10` minutes.
+- **caSecretName** - Name of the secret which contains the root Certificate Authority (CA). The default value is `nginx-auth-ca`.
```
Comment (franpog859) [inline resources/application-connector/charts/connector-service/README.md]: If I understand it correctly and this `README.md` file provides information about chart parameters, and `components/connector-service/README.md` file provides information about arguments from `options.go` file then this default value should be `connector-service-app-ca`

File: resources/application-connector/charts/connector-service/README.md [hunk 2]
```diff
@@ -12,14 +14,16 @@ The Connector Service has the following parameters, that can be set through the
 - **tokenLength** - Length of registration tokens. The default value is `64`.
 - **appTokenExpirationMinutes** - Time after which tokens for applications expire and are no longer valid. The default value is `5` minutes.
 - **runtimeTokenExpirationMinutes** - Time after which tokens for runtimes expire and are no longer valid. The default value is `10` minutes.
-- **caSecretName** - Name of the secret which contains the root Certificate Authority (CA). The default value is `nginx-auth-ca`.
+- **caSecretName** - Name of the secret which contains the root Certificate Authority (CA). The default value is `connector-service-app-ca`.
```
Comment (tomekpapiernik) [inline resources/application-connector/charts/connector-service/README.md]:
```suggestion
- **caSecretName** - Name of the Secret which contains the root Certificate Authority (CA). The default value is `connector-service-app-ca`.
```

File: resources/application-connector/charts/connector-service/README.md [hunk 3]
```diff
@@ -12,14 +14,16 @@ The Connector Service has the following parameters, that can be set through the
...
-- **gatewayHost** - Host at which gateway service is accessible The default value is `gateway.wormhole.cluster.kyma.cx`.
-- **certificateProtectedHost** - Host secured with client certificate, used for certificate renewal. The default host is `gateway.wormhole.cluster.kyma.cx`.
+- **appRegistryHost** - Host under which the Application Registry is accessible. The default value is an empty string.
+- **eventsHost** - Host under which the Event Service is accessible. The default value is an empty string.
 - **appsInfoURL** - URL at which management information for applications is available. If not provided, it bases on `connectorServiceHost`.
 - **runtimesInfoURL** - URL at which management information for runtimes is available. If not provided, it bases on `connectorServiceHost`.
-- **certificateValidityTime** - Validity time of certificates issued by this service. The default value is 90 days.
+- **group** - Group for which certificates are generated. If the chart does not provide the default value, you must specify it in the request header to the token endpoint.
```
Comment (tomekpapiernik) [inline resources/application-connector/charts/connector-service/README.md]:
```suggestion
- **group** - Group for which certificates are generated. If the chart does not provide a default value, you must specify it in the request header of the request sent to the token endpoint.
```

File: resources/application-connector/charts/connector-service/README.md [hunk 4]
```diff
@@ -12,14 +14,16 @@ The Connector Service has the following parameters, that can be set through the
...
+- **tenant** - Tenant for which certificates are generated. If the chart does not provide the default value, you must specify it in the request header to the token endpoint.
```
Comment (tomekpapiernik) [inline resources/application-connector/charts/connector-service/README.md]:
```suggestion
- **tenant** - Tenant for which certificates are generated. If the chart does not provide the default value, you must specify it in the header of the request sent to the token endpoint.
```

File: resources/application-connector/charts/connector-service/README.md [hunk 5]
```diff
@@ -12,14 +14,16 @@ The Connector Service has the following parameters, that can be set through the
...
+- **certificateValidityTime** - Time after which certificates issued by Connector Service will expire. Allowed time units are: `d`, `h`, `m` for days, hours and minutes, only single time unit can be used. Default value is `92d`.
```
Comment (tomekpapiernik) [inline resources/application-connector/charts/connector-service/README.md]:
```suggestion
- **certificateValidityTime** - Time after which the certificates issued by the Connector Service expire. Allowed time units are: `d`, `h`, `m` for days, hours and minutes. You can use only one unit at a time. The default value is `92d`.
```

**Pull request merged**